rustls/src/lib.rs

```diff
@@ -3,6 +3,8 @@
 //! <!-- TODO(portable-rustls) CLEANUP & IMPROVE NOTE FOR THIS FORK -->
 //! IMPORTANT NOTICE: REGARDLESS OF UPSTREAM `rustls` PROJECT THIS FORK IS NOT CERTIFIED AND NOT PEER-REVIEWED - USE AT YOUR OWN RISK AS STATED FURTHER BELOW
 //!
+//! XXX TODO MSRV FOR THIS FORK (XXX TODO AS WILL BE DOCUMENTED FURTHER BELOW)
```

Comment:
As I said in another comment: can remove from here now that there is MSRV badge near the beginning of this README. MSRV badge may be out of sync between releases but removing this would mean one less thing to keep in sync moving forward. Should also use `admin/pull-readme` to remove corresponding MSRV comment from README.